packages/ui/components/input-stepper/src/LionInputStepper.js

```diff
@@ -36,6 +36,11 @@ export class LionInputStepper extends LocalizeMixin(LionInput) {
         type: Number,
         reflect: true,
       },
+      valueText: {
+        type: Number,
```

Comment:
Since the stepper is always about numbers, do we really need to extend the API with such property? Does it make sense to just set `aria-valuenow` and `aria-valuetext` with the same value internally?

Reply:
ValueText should be a string :)

Reply:
Right :) But main question was why having this property at all? Is having number for `aria-valuetext` not enough? Like `aria-valuetext="2"`

Reply:
> The aria-valuenow property is required for the meter, scrollbar, separator (if the element is focusable), and slider roles.

> When the element's values are contained within a range but those values are not numeric (such as "small", "medium" and "large"), or they are numeric but there is value in communicating more information than just a number, aria-valuetext is used to surface the text value to assistive technologies. Only use aria-valuetext when aria-valuenow is not sufficiently meaningful for users because using aria-valuetext will prevent assistive technologies from communicating aria-valuenow.
>
> For example, for a battery indicator, it would be useful to know how many minutes are remaining, in addition to the percentage of charge remaining.
>
> ```html
> <span id="battery-label">Battery</span>
> <span role="meter" aria-labelledby="battery-label"
>   aria-valuenow="5"
>   aria-valuetext="5%, 18 minutes remaining.">
> </span>

https://www.w3.org/WAI/ARIA/apg/practices/range-related-properties/